synthtool/gcp/common.py

```diff
@@ -217,3 +217,10 @@ def decamelize(value: str):
         "([A-Z]+)([A-Z])([a-z0-9])", r"\1 \2\3", str_decamelize
     )  # ACLBatman -> ACL Batman.
     return re.sub("([a-z0-9])([A-Z])", r"\1 \2", str_decamelize)  # FooBar -> Foo Bar.
+
+
+def load_repo_metadata(metadata_file: str = "./.repo-metadata.json") -> Optional[Dict]:
```

Comment:
Should this be `_load_repo_metadata()`?

Reply:
seems like a reasonable change to make, if not too much of a hassle.